configure.ac

```diff
@@ -1032,6 +1032,14 @@ if test "${enable_gc_assertions}" = yes; then
               [Define to enable internal debug assertions.])
 fi
 
+AC_ARG_ENABLE(valgrind-tracking,
+    [AS_HELP_STRING([--enable-valgrind-tracking],
+        [heap profiler allocation tracking])])
+if test "${enable_valgrind_tracking}" = yes; then
+    AC_DEFINE([VALGRIND_TRACKING], 1,
+              [Define to enable heap profiler allocation tracking.])
```

Comment:
I think it is better to have the message matching that in cmake script, e.g.: Define to support tracking ...

Reply:
Good point, fixed.